src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java

```diff
@@ -48,7 +31,8 @@
     private final Set<String> entityTagFields;
     // adding alias,account_alias,cluster_alias due to legacy, and should be exclusive anyways
     private final static Set<String> DEFAULT_ENTITY_TAG_FIELDS = new HashSet<>(
-        Arrays.asList("application_id", "application_version", "stack_name", "stack_version", "application","version","account_alias","cluster_alias","alias"));
+        Arrays.asList("application_id", "application_version", "stack_name", "stack_version", "application",
```

Comment:
If this is constant (looks like), ImmutableSet.of(<varargs>) would be more appropriate here.

Reply:
Done